docs/tests/fixtures/0010_wrong_type_field_dict.json

```diff
@@ -0,0 +1,6 @@
+{
```

Comment:
any reason the number in the file name goes from 0004 to 0010? (also some numbers are similar)

Reply:
I've tried to work using "blocks" of similar tests and "space" them to make sure I could squeeze more tests in the blocks without having to rename all my fixtures.
The numbers are following the schema definition logic, from the most basic to the most complicated.
On top of that, they haven't been written in order, I've jumped from one idea to another, so I've put "spaces" when I was starting to write a test that was a bit ahead, so sometimes there's a gap between fixture numbers.

Reply:
Thanks for the answer :)